### PR TITLE
Deployment payload processing

### PR DESCRIPTION
**Pull request by gjtempleton**

### CODE REVIEW

**Comment by gnhenderson** [comment]
added some ignorable comments, otherwise LGTM into Skeleton

File: slingshotapi/service.py [hunk 1]
```diff
@@ -79,6 +97,57 @@ def _save_service(cls, service_data):
...
-        return []
+    def _process_parameter(param):
+        """
+        :param param: The parameter variable to be inserted into the versions
+        :return: a string representation compatible with json
+        """
+        if type(param) == str:
+            return '"{}"'.format(param)
+        elif type(param) == bool:
+            return str(param).lower()
```
Comment (gjtempleton) [inline slingshotapi/service.py]: Good spot, got too delete happy on the refactor. Reinstated the relevant line. Tests are now failing as I'm halfway through the refactor for moving the loaders out into fixtures.

**Comment by gjtempleton** [comment]
Now addressed all of your comments I think, and tests are passing again.

**Comment by gnhenderson** [comment]
LGTM

**Pull request merged**